### ISSUE TITLE
Accessibility issue related HTML element

### ISSUE THREAD

**Issue by keuller** [BuilderIO/qwik#1983]
### Qwik Version

0.12.1

### Operating System (or Browser)

OSX

### Node Version (if applicable)

16

### Which component is affected?

Qwik City

### Expected Behaviour

Add `lang` attribute on `html` tag. 

### Actual Behaviour

`html` tag element does not have `lang` attribute. The only way to add it is on `body` tag which means it's not a standard and affects PageSpeed Accessibility indicator.

### Additional Information

As you can see PageSpeed complains about it.

![Screenshot 2022-11-04 at 09 25 27](https://user-images.githubusercontent.com/2074316/199938588-ff0f253b-c3aa-4463-81c4-f58bfbf4f126.png)

I tried to identify where Qwik generate `html` in order to contribute with fix, but I won't be able to find it on the source code. If anyone help me I can fix it.

**Comment by nnelgxorz** [comment]
Hey @keuller! You can set the lang attribute (and other attributes) on the html tag using the containerAttributes field on the RenderToStreamOptions. You can add it in your `entry.ssr.tsx`.

```typescript
export default function (opts: RenderToStreamOptions) {
  return renderToStream(<Root />, {
    manifest,
    ...opts,
    containerAttributes: {
      lang: 'en'
    },
    prefetchStrategy: {
      ///....
    },
  });
}
```

I think we should add this to the starters because the question comes up a lot.

**Comment by keuller** [comment]
@nnelgxorz Thank you a lot, it works. It would be nice add this information on Docs.

**Comment by phcoliveira** [comment]
Is there a way to use the `RequestEvent` at this point?
It would be nice to read a cookie or a URL in order to change the `lang` attribute.
Is it possible for a route (or its action) to set a property of `RenderToStreamOptions`?
